src/libphonenumber/PhoneNumber.php

```diff
@@ -33,50 +56,101 @@ public function clearCountryCode()
     }
 
     /**
-     * @var int
+     * The national number.
+     *
+     * @var string|null
```

Comment:
I think there was a confusion on the type of *national number*: tests indeed [set it as an integer](https://github.com/giggsey/libphonenumber-for-php/blob/master/Tests/libphonenumber/Tests/core/PhoneNumberTest.php#L63), but `PhoneNumberUtil` [sets it as a string](https://github.com/giggsey/libphonenumber-for-php/blob/master/src/libphonenumber/PhoneNumberUtil.php#L1505).